fili-core/src/main/java/com/yahoo/bard/webservice/data/config/table/ConcretePhysicalTableDefinition.java

```diff
@@ -53,7 +78,48 @@ public ConcretePhysicalTableDefinition(
             Set<? extends DimensionConfig> dimensionConfigs,
             Map<String, String> logicalToPhysicalNames
     ) {
-        super(name, timeGrain, metricNames, dimensionConfigs, logicalToPhysicalNames);
+        super(
+                name,
```

Comment:
Make a note in the documentation for this method that it defaults the table to having no configured start or end date.